assessment.py

```diff
@@ -129,16 +131,52 @@ def POST(self):
         try:
             am = autils.get_assessment_manager()
             form = am.get_bank_form_for_create([])
+            data = self.data()
 
-            form = utilities.set_form_basics(form, self.data())
+            form = utilities.set_form_basics(form, data)
 
             new_bank = utilities.convert_dl_object(am.create_bank(form))
 
+            if 'aliasId' in data:
+                am.alias_bank(utilities.clean_id(json.loads(new_bank)['id']),
+                              utilities.clean_id(data['aliasId']))
+
             return new_bank
         except (PermissionDenied, InvalidArgument) as ex:
             utilities.handle_exceptions(ex)
 
 
+class AssessmentRemoveAssignedBankIds(utilities.BaseClass):
+    """Remove a single bankId from the list of assignedBankIds"""
+    @utilities.format_response
+    def DELETE(self, bank_id, assessment_id, assigned_bank_id):
+        try:
+            am = autils.get_assessment_manager()
+            assigned_bank_id = am.get_bank(utilities.clean_id(assigned_bank_id)).ident
+            am.unassign_assessment_from_bank(utilities.clean_id(assessment_id),
+                                             assigned_bank_id)
+            return web.Accepted()
+        except (PermissionDenied, IllegalState, InvalidId, OperationFailed) as ex:
+            utilities.handle_exceptions(ex)
+
+
+class AssessmentAssignedBankIds(utilities.BaseClass):
```

Comment:
Um, right now it's random, I guess. We don't have an established convention or pattern for it -- happy to follow one that you might recommend.